addon/scripting_nodes/src/features/nodes/categories/Data/String/node_remove_whitespaces_in_string.py

```diff
@@ -0,0 +1,37 @@
+from scripting_nodes.src.features.nodes.base_node import ScriptingBaseNode
+import bpy
+
+
+class SNA_Node_Remove_Whitespaces_In_String(ScriptingBaseNode, bpy.types.Node):
+    bl_idname = "SNA_Node_Remove_Whitespaces_In_String"
+    bl_label = "Remove Whitespaces In String"
```

Comment:
I think we can remove the "In String" part from this name just to keep the names a bit shorter